allennlp/nn/util.py

```diff
@@ -55,6 +56,9 @@ def sort_batch_by_length(tensor: torch.autograd.Variable, sequence_lengths: torc
     restoration_indices : Variable(torch.LongTensor)
         Indices into the sorted_tensor such that
         ``sorted_tensor.index_select(0, restoration_indices) == original_tensor``
+    permuation_index : Variable(torch.LongTensor)
```

Comment:
Shouldn't this be `indices`, to match `restoration_indices`?  Also, a comment like the `sorted_tensor.index_select(...` comment above would be nice here.